File: C++/Bit Manipulation/136. Single Number.cpp
/* 136. Single Number
Difficulty: Easy
Submission Link: https://leetcode.com/problems/single-number/submissions/1386409830/ */

class Solution {
public:
    int singleNumber(vector<int>& nums) {
      int x=0;
            for(int i=0;i<nums.size();i++){
                    x = x^nums[i];
            }
            return x;
    }
};

File: C#/Bit Manipulation/136. Single Number.cs
/* 136. Single Number
Difficulty: Easy
Submission Link: https://leetcode.com/problems/single-number/submissions/1386409966/ */

public class Solution {
    public int SingleNumber(int[] nums) {
        int ans = 0;
        foreach(int x in nums)
            ans ^= x;
        return ans;
    }
}